jest.setup.mjs

```diff
@@ -1 +1,5 @@
+import { TextEncoder, TextDecoder } from 'util';
 import '@testing-library/jest-dom/extend-expect';
+
+global.TextEncoder = TextEncoder;
```

Comment:
I honestly don't like to simply inject Node's TextEncoder API to the `globals`. That's a hack and shouldn't be necessary.

You can simply mock `isomorphic-dompurify` and that will have the same effect.

Since there's nothing to be "purified" on our unit-test. (We don't have malicious code)